Analytics integration news (#436)
configcat/docs: website/docs/news.mdx

{/* This page must be in sync with the news page on the Dashboard: https://github.com/configcat/app/blob/master/src/app/home-module/news/news.component.html */}

## Advanced feature flag analytics with Mixpanel, Amplitude and Google Analytics

#### May 15, 2024

> We're happy to share that we have introduced new ways to track your featuer flags in your Analytics tools.

Reviewer: ```suggestion
We're happy to share that we have introduced new ways to track your feature flags in your analytics tools.

From now on you can:
```

not sure about the analytics one

 <ul>
    <li>
        <a href="https://configcat.com/docs/integrations/mixpanel/#annotations" target="_blank"

>             rel="noopener noreferrer">Monitoring your feature flag change events in Mixpanel with Annotations</a>

Reviewer: ```suggestion
            rel="noopener noreferrer">Monitor feature flag change events in Mixpanel with Annotations</a>
```

    </li>
    <li>
        <a href="https://configcat.com/docs/integrations/mixpanel/#experiments" target="_blank"

>             rel="noopener noreferrer">Sending feature flag evaluation analytics to Mixpanel Experiments</a>

Reviewer: ```suggestion
            rel="noopener noreferrer">Send feature flag evaluation analytics to Mixpanel Experiments</a>
```

    </li>
    <li>
        <a href="https://configcat.com/docs/integrations/amplitude/#experiments" target="_blank"

>             rel="noopener noreferrer">Sending feature flag evaluation analytics to Amplitude Experiments</a>

Reviewer: ```suggestion
            rel="noopener noreferrer">Send feature flag evaluation analytics to Amplitude Experiments</a>
```

    </li>
    <li>
        <a href="https://configcat.com/docs/integrations/google-analytics" target="_blank"

>             rel="noopener noreferrer">Sending feature flag evaluation analytics to Google Analytics</a>

Reviewer: ```suggestion
            rel="noopener noreferrer">Send feature flag evaluation analytics to Google Analytics</a>
```
